src/services/vcmap/context.ts

```diff
@@ -0,0 +1,73 @@
+import {
+  CesiumMap,
+  CesiumTilesetLayer,
+  MapCollection,
+  OpenlayersMap,
+  OpenStreetMapLayer,
+  TerrainLayer,
+  Viewpoint,
+  WMSLayer,
+} from '@vcmap/core'
+
+const layers = [
+  new OpenStreetMapLayer({ name: 'osmBase', activeOnStartup: true }),
+  new TerrainLayer({
+    name: 'terrain',
+    url: 'https://demo.virtualcitymap.de/rennes/datasource-data/b3ef17bf-fdde-4979-8f05-8b4db5811c43',
+    activeOnStartup: true,
+  }),
+  new CesiumTilesetLayer({
+    name: 'building',
+    url: 'https://demo.virtualcitymap.de/rennes/datasource-data/f661c55d-d40b-44fb-889f-88176163cba2',
+    activeOnStartup: true,
+  }),
+  new WMSLayer({
+    name: 'metro',
+    url: 'https://public.sig.rennesmetropole.fr/geoserver/ows',
+    layers: 'trp_org:sd_velo_iti_2018',
+    activeOnStartup: false,
+    parameters: {
+      transparent: true,
+    },
+  }),
+  new WMSLayer({
+    name: 'bus',
+    url: 'https://public.sig.rennesmetropole.fr/geoserver/ows',
+    layers: 'trp_org:sd_velo_iti_2018',
+    activeOnStartup: false,
+    parameters: {
+      transparent: true,
+    },
+  }),
+  new WMSLayer({
+    name: 'bike',
+    url: 'https://public.sig.rennesmetropole.fr/geoserver/ows',
+    layers: 'trp_org:sd_velo_iti_2018',
```

Comment:
No `activeOnStartup: false` also?

Reply:
yes, anyway it's the default value. And event putting it to true is not adding the layer on startup...